src/include/planner/plannodes/plan_node_defs.h

```diff
@@ -141,9 +141,12 @@ enum class SetOpType { INVALID = INVALID_TYPE_ID, INTERSECT = 1, INTERSECT_ALL =
 using IndexExpression = common::ManagedPointer<parser::AbstractExpression>;
 /** Type of index scan. */
 enum class IndexScanType : uint8_t {
+  Dummy,
```

Comment:
If I'm understanding `Dummy` correctly then it may help to add a short comment saying something along the lines that `Dummy` is a placeholder value and shouldn't actually be used as a scan type.

Reply:
Updated